client/lib/abtest/index.js

```diff
@@ -100,17 +134,34 @@ ABTest.prototype.getVariation = function() {
 };
 
 ABTest.prototype.isEligibleForAbTest = function() {
-	var selectedSite = sites.getSelectedSite();
+	const selectedSite = sites.getSelectedSite();
+	const client = ( typeof navigator !== 'undefined' ) ? navigator : {};
```

Comment:
We're only doing this to avoid a `ReferenceError`, so it might be clearer to shadow `navigator` rather than introducing a new variable.